usage.py

```diff
@@ -4,13 +4,18 @@
 import dash_html_components as html
 import dash_core_components as dcc
 import plotly.graph_objs as go
+import dash_daq as daq
+
 
 app = dash.Dash(__name__)
 
 app.scripts.config.serve_locally = True
 app.css.config.serve_locally = True
+app.config['suppress_callback_exceptions']=True
 
 app.layout = html.Div([
+    daq.BooleanSwitch(id='toggle-drag', on=True),
```

Comment:
Nice use of DAQ here! Could you add it to `requirements.txt` so that people using `pip install -r requirements.txt` could have it installed as well?

Reply:
All done also added core components for kix if needed later